### PR TITLE
Debian test fixes

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Few changes that allow running tests on Debian.

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

File: tests/utils.py [hunk 1]
```diff
@@ -209,6 +209,27 @@ def run_command(command):
     out, err = res.communicate()
     return (res.returncode, out.decode().strip(), err.decode().strip())
```
Comment (vpodzime) [inline tests/utils.py]: *ditribution* in the commit message

File: tests/crypto_test.py [hunk 2]
```diff
@@ -353,7 +353,7 @@ def setUp(self):
         self.addCleanup(os.unlink, self.public_cert)
 
     @unittest.skipIf("SKIP_SLOW" in os.environ, "skipping slow tests")
-    @skip_on(("centos", "enterprise_linux"))
+    @skip_on(("centos", "enterprise_linux", "debian"))
```
Comment (vpodzime) [inline tests/crypto_test.py]: Please add the ``reason`` argument here.

File: tests/kbd_test.py [hunk 3]
```diff
@@ -56,6 +56,7 @@ def _clean_up(self):
         if self._loaded_zram_module:
             os.system("rmmod zram")
 
+@skip_on("debian", reason="loading zram module is broken on Debian")
```
Comment (vpodzime) [inline tests/kbd_test.py]: Why don't you add this to the base class?

**Pull request merged**